jep/0000/README.adoc

```diff
@@ -0,0 +1,227 @@
+= JEP-0000: Readonly system configuration
+:toc: preamble
+:toclevels: 3
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+
+.Metadata
+[cols="1h,1"]
+|===
+| JEP
+| 0000
+
+| Title
+| Readonly system configuration
+
+| Sponsor
+| link:https://github.com/timja[timja]
+
+// Use the script `set-jep-status <jep-number> <status>` to update the status.
+| Status
+| Not Submitted :information_source:
+
+| Type
+| Standards
+
+| Created
+| 2020-01-20
+
+| BDFL-Delegate
+| link:https://github.com/oleg-nenashev[oleg-nenashev]
+
+//
+//
+// Uncomment if there is an associated placeholder JIRA issue.
+| JIRA
+| https://issues.jenkins-ci.org/browse/JENKINS-12548[JENKINS-12548]
+//
+//
+// Uncomment if discussion will occur in forum other than jenkinsci-dev@ mailing list.
+//| Discussions-To
+//| :bulb: Link to where discussion and final status announcement will occur :bulb:
```

Comment:
Would be great to kick-off the dev list discussion

Reply:
Attempting to kickstart this discussion on the dev mailing lists --->  https://groups.google.com/d/msg/jenkinsci-dev/SHWZiMfbPEs/ZZEj00OsEAAJ